terraform/services/api-waf-sync/main.tf

```diff
@@ -22,10 +22,10 @@ module "api_waf_sync_function" {
   schedule_expression = "cron(0/10 * * * ? *)"
 
   environment_variables = {
-    ENV             = var.env
-    APP_NAME        = "dpc-${var.env}-api-waf-sync"
-    WAF_IP_SET_NAME = "DPC_${upper(var.env)}_Implementer_IP_Set"
-    DB_HOST         = data.aws_ssm_parameter.dpc_db_host.value
+    ENV      = var.env
+    APP_NAME = "${var.app}-${var.env}-api-waf-sync"
+    DB_HOST  = "data.aws_ssm_parameter.dpc_db_host.value"
+    DB_URL   = "data.aws_ssm_parameter.bcda_db_url.value"
```

Comment:
Possible to pick one of DB_HOST and DB_URL and just make one data call?

Reply:
DPC has a bit more complicated a route into their DB then BCDA seems to and our parameter stores have different existing variables.  Could be nice to simplify that at some point?

Reply:
Yeah okay I'm fine with punting on it for now.